packages/server/assets/core/typedefs/apitoken.graphql

```diff
@@ -55,6 +55,28 @@ input AppTokenCreateInput {
   limitResources: [TokenResourceIdentifierInput!]
 }
 
+"""
+A token used to enable an embedded viewer for a private project
+"""
+type EmbedToken {
+  tokenId: String!
+  projectId: String!
+  userId: String!
+  modelIds: String!
+  createdAt: DateTime!
+  lifespan: BigInt!
+  lastUsed: DateTime!
```

Comment:
I'm wondering if the token itself shouldnt be a prop here, or at the very least a partially obfuscated version. Cause we'll need to list these in the FE at some point.

Maybe if its the real secret token, we add checks that only allow showing it in a non obfuscated version to project owners, and everyone else sees asterisks?

Reply:
Discussed with Mike and will hand-wave that this is enough (with inclusion of your `LimitedUser` change)